drivers/driver-test/src/commonMain/kotlin/com/squareup/sqldelight/driver/test/DriverTest.kt

```diff
@@ -38,33 +38,33 @@ abstract class DriverTest {
     }
 
     override fun migrate(
-      db: SqlDatabase,
+      db: SqlDriver,
```

Comment:
Should we also change the name here?

Reply:
gonna leave for now just cause it went green 😅